### PR TITLE
First launch of the new homepage

### PR DESCRIPTION
**Pull request by eronisko**
This replaces the old "index" page with the new one.

"slide" clicks are now being tracked via Livewire

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/views/livewire/home/tracked-featured-piece-button.blade.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<a wire:click.prevent="trackClick" href="{{ $featuredPiece->url }}"
+    class="{{ $class }} tw-inline-block tw-border tw-border-gray-300 tw-px-4 tw-py-2 tw-text-sm tw-transition tw-duration-300 hover:tw-border-gray-400 hover:tw-bg-white hover:tw-text-gray-800">
+    {{ $featuredPiece->is_collection ? 'Prejsť na kolekciu' : 'Prejsť na článok' }}
```
Comment (igor-kamil) [inline resources/views/livewire/home/tracked-featured-piece-button.blade.php:3]: before release to production, I expected to add missing translations

**Review by igor-kamil — APPROVED** [review]
✨

File: resources/lang/en/home.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    'from_galleries_end' => 'Slovak and Czech galleries',
+    'from_authors_start' => 'by',
+    'from_authors_end' => 'artists',
+    'in_high_res_start' => 'of which',
+    'in_high_res_end' => 'in high resolution',
+    'are_free_start' => 'of which',
+    'are_free_end' => 'in public domain',
+    'show_more_articles' => 'more articles',
+
+    'featured_piece' => [
```
Comment (eronisko) [inline resources/lang/en/home.php:16]: @ustengrav

File: resources/lang/cs/home.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    'from_galleries_end' => 'slovenských a českých galerií',
+    'from_authors_start' => 'od',
+    'from_authors_end' => 'autorů',
+    'in_high_res_start' => 'z nichž',
+    'in_high_res_end' => 've vysokém rozlišení',
+    'are_free_start' => 'z nichž',
+    'are_free_end' => 'autorsky volných',
+    'show_more_articles' => 'více článků',
+
+    'featured_piece' => [
```
Comment (eronisko) [inline resources/lang/cs/home.php:16]: @ustengrav

**Review by eronisko — COMMENTED** [review]
Ready for translations

**Pull request merged**